source/elements/oneDAL/source/glossary.rst

```diff
@@ -0,0 +1,277 @@
+        :term:`dataset` with a goal to make predictions of :term:`responce`
+        values based on input :term:`feature vectors<Feature vector>`.
+
+        **Synonym:** prediction
+
+    Inference set
+        A :term:`dataset` (usually without :term:`responses<Response>` present)
+        used at the :term:`inference` stage.
+
+    Interval feature
+        A :term:`continuous feature` which values can be compared, added or
+        subtracted, but cannot be multiplied or divided.
+
+        **Examples:** a timeframe scale, a temperature in Celcius or Fahrenheit
+
+    Label
+        A :term:`response` with :term:`categorical<Categorical feature>` or
+        :term:`ordinal<Ordinal feature>` values. This is an output in
+        :term:`classification` and :term:`clustering` tasks.
+
+        **Example:** the spam-detection task has a binary label indicating whether
+        the email is spam or not
+
+    Model
+        An entiry which stores an information necessary to run :term:`inference`
+        on a new :term:`dataset`. Typically a result of :term:`training`
+        process.
+
+        **Example:** in linear regression algorithm, the model contains weight
+        values for each input feature and single bias value
+
+    Observation
+        A :term:`feature vector` and zero or more :term:`responses<Response>`.
+        Each of them can encode different objects or events.
+
+        **Synonyms:** instance, sample
+
+    Ordinal feature
+        A :term:`categorical feature` with defined operations of equality an
+        ordering between values.
+
+        **Example:** student's grade
+
+    Ratio feature
+        A :term:`continuous feature` with defined operations of equality,
+        comparison, addition, substraction, multiplication and division.
+        Has a zero value element which means the absence of value.
+
+        **Example:** the height of a tower
+
+    Regression
+        A :term:`supervised machine-learning problem<Supervised learning>` of
+        assigning :term:`continuous<Continuous feature>`
+        :term:`responses<Response>` to :term:`feature vectors<Feature vector>`.
+
+        **Example:** prediction of the temperature based on weather conditions
+
+    Response
+        A property of some real object or event which value is target to be
+        predicted in the :term:`supervised learning` task. While the
+        :term:`feature` is an input in the machine learning task, the response
+        is one of the outputs can be made by the :term:`model` on the
+        :term:`inference` stage.
+
+        **Synonym:** dependent variable
+
+    Supervised learning
+        :term:`Training` process with a :term:`dataset` containing information
+        on dependencies between :term:`features<Feature>` and
+        :term:`responses<Response>`. The goal is to get a :term:`model` of
+        dependencies between input :term:`feature vector` and
+        :term:`responses<Response>`.
```

Comment:
```suggestion
        :term:`Training` process that uses a :term:`dataset` with information
        about dependencies between :term:`features <Feature>` and
        :term:`responses <Response>`. The goal is to get a :term:`model` of
        dependencies between input :term:`feature vector` and
        :term:`responses <Response>`.
```

Also note that here the reference goes to `dataset`, while in unsupervised learning you refer `training set`.